apps/frontend-manage/src/components/resources/SharedAnswerCollectionList.tsx

```
@@ -0,0 +1,23 @@
+import { AnswerCollection } from '@klicker-uzh/graphql/dist/ops'
+import { useTranslations } from 'next-intl'
+import AnswerCollectionCollapsible from './answerCollections/AnswerCollectionCollapsible'
+
+function SharedAnswerCollectionList({}: {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)